packages/core/src/filters/Filter.ts

```diff
@@ -297,6 +327,13 @@ export class Filter extends Shader
 
     set resolution(value: number)
     {
+        this._autoResolution = false;
+
+        if (this._resolution === value)
+        {
+            return;
+        }
+
```

Comment:
We don't really need this bail-out, since it's only re-assigning a number.

```suggestion
        this._autoResolution = false;
```